CalendarKit-Swift/SwiftCal/SwiftCal.swift

```diff
@@ -60,6 +61,14 @@ public class Read {
 
         if calendarEvents.count > 0 {
 
+            var methodString: NSString?
+            let methodScanner = Scanner(string: icsString)
+            methodScanner.scanUpTo("METHOD:", into: nil)
+            methodScanner.scanUpTo("\r\n", into: &methodString)
+            if let theMethodString = methodString {
+                calendar.method = String(theMethodString.substring(from: 7))
```

Comment:
aren't we already scanning up to `METHOD:`, why do we need `substring(from: 7))` then?

Reply:
Oh nvm, I see we're only scanning that to nil to test if it exists?

Reply:
It will scan up to the begging of "METHOD:", but we really want the string starting after the colon.